core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2051,6 +2051,39 @@ package object config {
       .doubleConf
       .createWithDefault(0.75)
 
+  private[spark] val SPECULATION_TASK_MIN_DURATION =
+    ConfigBuilder("spark.speculation.task.min.duration")
```

Comment:
I'm quite confused about these three configs:

```
spark.speculation.task.min.duration
spark.speculation.minTaskRuntime
spark.speculation.task.duration.threshold
```

Seems like the previous two are equivalent.